core/app/controllers/brightcontent/sessions_controller.rb

```diff
@@ -21,7 +21,19 @@ def create
 
     def destroy
       session[:brightcontent_user_id] = nil
+      session[:return_to] = nil
```

Comment:
Yes, I think that is good practice (and also after a login, to avoid session hijacking - e.g. when a session id (without the secure bit set) is present and the user comes via http, the session id is exposed; then when you login, if the session is not reset, the attacker can use the eavesdropped session id to gain access; perhaps there are other scenario's). Didn't include that because it's a distinct feature.
I'd like to keep it this way and tackle the session reset in a different PR.